doc/API.md

```diff
@@ -278,14 +289,73 @@ feerate.
 | ---------- | ------ | ----------------------------------------------- |
 | `spend_tx` | string | Base64-encoded Spend transaction PSBT           |
 
-### `spendtx`
+
+### `updatespendtx`
+
+The `updatespendtx` RPC Command stores or update the stored Spend transaction with the
+given one.
+
+#### Request
+
+| Field       | Type         | Description                                                           |
+| ----------- | ------------ | --------------------------------------------------------------------- |
+| `spend_tx`  | string       | Base64-encoded Spend transaction PSBT                                 |
+
+#### Response
+
+None; the `result` field will be set to the empty object `{}`. Any value should be
+disregarded for forward compatibility.
+
+
+### `delspendtx`
+
+#### Request
+
+| Field          | Type   | Description                                         |
+| -------------- | ------ | --------------------------------------------------- |
+| `spend_txid`   | string | Hex encoded txid of the Spend transaction to delete |
+
+#### Response
+
+None; the `result` field will be set to the empty object `{}`. Any value should be
+disregarded for forward compatibility.
+
+
+### `listspendtxs`
+
+#### Request
+
+| Field          | Type   | Description                                         |
+| -------------- | ------ | --------------------------------------------------- |
+
+#### Response
+
+| Field          | Type   | Description                                                          |
+| -------------- | ------ | -------------------------------------------------------------------- |
+| `spend_txs`    | array  | Array of [Spend transaction resources](#spend_transaction_reources)  |
+
+##### Spend transaction resources
+
+| Field                  | Type          | Description                                                           |
+| ---------------------- | ------------- | --------------------------------------------------------------------- |
+| `deposit_outpoints`    | string array  | Array of the deposit outpoints of the vaults this transaction spends  |
+| `psbt`                 | string        | Base64-encoded Spend transaction PSBT                                 |
```

Comment:
Is it possible to modify `getspendtx` response in this PR, in order to have the same `Spend transaction resource` for both `listspendtxs` and `getspendtx` ?

Reply:
Discussed in face to face and we agreed it was ugly to return something you've been input'ed